src/Nethermind/Nethermind.Trie/RustVerkleLib.cs

```diff
@@ -63,9 +63,9 @@ static RustVerkleLib()
         [DllImport("rust_verkle")]
         private static extern byte verify_verkle_proof_multiple(IntPtr verkleTrie, byte[] verkleProof, int proof_len, byte[,] key, byte[,] value, int len);
 
-        public static IntPtr VerkleTrieNew()
+        public static IntPtr VerkleTrieNew(byte databaseScheme = 0, byte commitScheme = 0)
```

Comment:
don't hardcode this, give user the option to select which one to use

Reply:
So having some default might be fine, don't know what they mean though without proper labels.

Reply:
I haven't hardcoded the values. The API invoker can pass the parameters as per their need. Is there any other file I need to update to give such option?

P.S: I will change byte to enum for better readability